Week1/Homework/create.js

```diff
@@ -0,0 +1,152 @@
+const util = require('util');
+const mysql = require('mysql');
+const connection = require('./connection');
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+async function seedDatabase() {
+  const CREATE_DATABASE = `CREATE DATABASE IF NOT EXISTS World`;
+  const USE_DATABASE = `USE World`;
+  const CREATE_COUNTRIES_TABLE = `
+    CREATE TABLE IF NOT EXISTS countries (
+      country_code INT NOT NULL,
+      country_name VARCHAR(50) NOT NULL,
+      country_continent VARCHAR(50) NOT NULL,
+      country_population FLOAT NOT NULL,
+      surface_areas FLOAT NOT NULL
+    );`;
+  const CREATE_CITIES_TABLE = `
+    CREATE TABLE IF NOT EXISTS cities (
+      city_name VARCHAR(20) NOT NULL,
+      city_population FLOAT NOT NULL,
+      city_code VARCHAR(3) NOT NULL
+    
+    );`;
+  const countries = [
+    {
+      country_code: '61',
+      country_name: 'Australia',
+      country_continent: 'Oceania & Australia',
+      country_population: '25,088,636',
+      surface_areas: '7,682,300',
+    },
+    {
+      country_code: '86',
+      country_name: 'China',
+      country_continent: 'Asia',
+      country_population: '1,420,062,022	',
+      surface_areas: '9,388,211',
+    },
+    {
+      country_code: '358',
+      country_name: 'Finland',
+      country_continent: 'Europe',
+      country_population: '5,561,389',
+      surface_areas: '303,890',
+    },
+    {
+      country_code: '33',
+      country_name: 'France',
+      country_continent: 'Europe',
+      country_population: '65,480,710',
+      surface_areas: '547,557',
+    },
+    {
+      country_code: '49',
+      country_name: 'Germany',
+      country_continent: 'Europe',
+      country_population: '82,438,639',
```

Comment:
I'm not sure - did the comma give you problem when converting to INT during insertion?